### PR TITLE
Explicitly label graphs for 4.X making labels uniq

### PR DESCRIPTION
**Pull request by pcarinhas**
Fixes ZEN-22543.

* Override getDefaultGraphDefs()
* Make all graph['title'] uniq

### CODE REVIEW

**Comment by cluther** [comment]
This isn't much different than just changing the titles of the graphs. The only difference is that it's more complicated, and it doesn't affect Zenoss 5.

I was thinking that you'd only be prefixing the title of graphs *not* coming from the super call. Those are the ones that need the extra context.

**Comment by pcarinhas** [comment]
We can end up with multiple VG's and PV's with Utilization. Without inserting self.title at that stage, you won't be able to identify which Utilization belongs to which device.

**Comment by pcarinhas** [comment]
Make changes as per comments.

File: ZenPacks/zenoss/LinuxMonitor/FileSystem.py [hunk 1]
```diff
@@ -85,10 +85,21 @@ def getRRDTemplateName(self):
...
-                graphs.append(graph)
+
+        # Explicitly re-label graphs displayed in other components for clarity
+        bd = self.logicalVolume() or self.blockDevice()
+        if bd:
+            for bdGraph in bd.getDefaultGraphDefs(drange):
+                if not bdGraph.get('title_override'):
+                    bdGraph = \
+                        {'title': '{0} ({1}: {2})'.format(bdGraph['title'],
+                                                         bd.__class__.__name__,
```
Comment (cluther) [inline ZenPacks/zenoss/LinuxMonitor/FileSystem.py]: We shouldn't expose literal class names to the user like this. I'd say either drop the type from the title, or use bd.class_label.

I have a style nitpick here too. Generally backslashes should be avoided unless they're really necessary to make the code read better. Indentation should also be consistent. I'd rather see this section look something like the following. It has advantages for keeping future diffs as small and precise as possible when this code needs updated.

```python
title = "{} ({}: {})".format(
    bdGraph["title"],
    bd.class_label,
    bd.titleOrId())

bdGraph = {
    "title": title,
    "url": bdGraph["url"],
    "title_override": True,
    }
```

**Comment by pcarinhas** [comment]
Refactored all getDefaultGraphsDefs() to use util.override_graph_labels()

**Pull request merged**